webhook/resources/backup/mutator.go

```diff
@@ -50,6 +53,35 @@ func (b *backupMutator) Create(request *admission.Request, newObj runtime.Object
 	}
 	patchOps = append(patchOps, patchOp)
 
+	backupLabels := make(map[string]string)
+	volumeName, isExist := backup.Labels[types.LonghornLabelBackupVolume]
+	if !isExist {
+		err := errors.Wrapf(err, "failed to ger volume name from backup label")
```

Comment:
```suggestion
		err := errors.Wrap(err, "failed to get volume name from backup label")
```

Reply:
Thank you. I will fix the typo and correct to use right function.